addons/images/rsu_ping_fetch/README.md

```diff
@@ -2,15 +2,19 @@
 
 ## Table of Contents
 
-- [About ](#about-)
-- [Requirements ](#requirements-)
+- [RSU Ping Fetch](#rsu-ping-fetch)
+  - [Table of Contents](#table-of-contents)
+  - [About ](#about-)
+  - [Requirements ](#requirements-)
 
 ## About <a name = "about"></a>
 
 This directory contains a microservice that runs within the RSU Manager GKE Cluster. The rsu_ping_fetch application populates the CV Manager PostGreSQL database's 'ping' table with the current online statuses of all RSUs recorded in the 'rsus' table. These statuses are retrieved directly from a [Zabbix server](https://www.zabbix.com/).

```

Comment:
Should we change this to say 'CV Manager' instead of 'RSU Manager'?

Reply:
Addressed